scripts/prepare-genesis/default.nix

```diff
@@ -16,7 +16,7 @@ with (import ./../../lib.nix);
 
 let
   yaml2json = haskell.lib.disableCabalFlag haskellPackages.yaml "no-exe";
-  genesisTools = [ yaml2json jq coreutils gnused cardano-sl-tools ];
+  genesisTools = [ yaml2json jq python3 coreutils gnused cardano-sl-tools ];
```

Comment:
why is python3 required? I thought it was just for yaml2json, which I replaced with a haskell version, but I could be mistaken.

Reply:
There is a `python -c` in the script to generate the random seed.